linera-service/src/client.rs

```diff
@@ -941,29 +941,31 @@ impl NodeService {
 #[allow(unused_mut)]
 fn detect_current_features() -> Vec<&'static str> {
     let mut features = vec![];
-    #[cfg(benchmark)]
+    // The "default" is required for the linera_indexer code.
+    features.push("default");
```

Comment:
How about removing `--no-default-features` below instead?

Reply:
I see the situation in that respect as not completely resolved.
But the right solution is unclear to me at this point.

Reply:
It seems to me that `linera-indexer` should require `rocksdb` not `default`. Let's see how the CI goes in https://github.com/linera-io/linera-protocol/pull/1030